locale/en/site.json

```diff
@@ -160,7 +160,7 @@
   "banners": {
     "index": {
       "startDate": "2022-06-27T16:00:00.000Z",
-      "endDate": "2021-07-13T16:00:00.000Z",
+      "endDate": "2022-07-13T16:00:00.000Z",
       "text": "New security releases to be made available July 5th, 2022",
       "link": "blog/vulnerability/july-2022-security-releases/"
```

Comment:
Looks like historically we've put full URL's here as the `link`. Presume the CI wasn't failing in the original PR as with the end date in the past (being corrected here) the link wasn't inserted into the homepage for it to be checked.